src/sage/combinat/ordered_tree.py

```diff
@@ -360,19 +360,20 @@ def to_parallelogram_polyomino(self, bijection=None):
         EXAMPLES::
 
             sage: T = OrderedTree([[[], [[], [[]]]], [], [[[],[]]], [], []])
-            sage: T.to_parallelogram_polyomino( bijection='Boussicault-Socci' )
-            [[0, 0, 0, 1, 1, 0, 1, 1, 1, 1, 0, 1, 1, 1], [1, 1, 1, 1, 1, 0, 0, 1, 1, 0, 1, 1, 0, 0]]
+            sage: T.to_parallelogram_polyomino(bijection='Boussicault-Socci')           # optional - sage.combinat
+            [[0, 0, 0, 1, 1, 0, 1, 1, 1, 1, 0, 1, 1, 1],
+             [1, 1, 1, 1, 1, 0, 0, 1, 1, 0, 1, 1, 0, 0]]
             sage: T = OrderedTree( [] )
-            sage: T.to_parallelogram_polyomino()
+            sage: T.to_parallelogram_polyomino()                                        # optional - sage.combinat
```

Comment:
top of file ?

Reply:
I'm shipping the trees from `sage.combinat` as part of **sagemath-graphs**; see https://deploy-preview-35749--sagemath-tobias.netlify.app/reference/misc/sage/features/sagemath.html#sage.features.sagemath.sage__graphs
So I want to be able to test these modules without having to pull in sagemath-combinat.
Only the doctests where it connects to other sorts of combinatorics (words, polyominos) are marked `# optional`